Cargo.toml

```diff
@@ -25,8 +25,8 @@ opt-level = 3
 
 [dependencies]
 
-egui = ">=0.20"
```

Comment:
![image](https://github.com/lucasmerlin/egui_skia/assets/5312813/47637386-e7e2-429b-8ccc-a5ef6b0cb3d6)

Dependency specs on this format seem ill-advised so I'm making it 0.24 precisely. Our project breaks on clean-builds without lockfiles now.

Perhaps future versions if compatible with 0.24 do `>=0.24<=0.2x`.

Perhaps the same thing should also be done with skia-safe, winit etc.

Reply:
Sounds good 👍